.github/workflows/update-filter-list.yml

```diff
@@ -0,0 +1,32 @@
+on:
+  workflow_dispatch:
+    schedule:
+      - cron: "0 0 * * SUN"
+
+jobs:
+  update-list:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Clone repository
+        uses: GuillaumeFalourd/clone-github-repo-action@v2
+        with:
+          owner: evilfactorylabs
+          repository: gatekeeper
+      - name: Set up git
+        run: |
+          echo ${{ github.actor }}
+          git config --global user.email "anakmagang@evilfactorylabs.com"
+          git config --global user.name "anakmagang"
+          # git config --global user.name github-actions
+          # git config --global user.email github-actions@github.com
```

Comment:
not sure ab this @faultables

Reply:
```suggestion
          git config --global user.email "ayang@evilfactorylabs.com"
          git config --global user.name "anakmagang"
```

Reply:
maksud gw kek push token or something like that